README.md

```diff
@@ -223,15 +223,15 @@ For visualizing the active zoneset in RViz you can run the `active_zoneset_node`
+| CI | [![CI-Melodic](https://github.com/PilzDE/psen_scan_v2/workflows/CI-Melodic/badge.svg?event=push)](https://github.com/PilzDE/psen_scan_v2/actions?query=event%3Apush+workflow%3ACI-Melodic+branch%3Amain) | [![CI-Noetic](https://github.com/PilzDE/psen_scan_v2/workflows/CI-Noetic/badge.svg?event=push)](https://github.com/PilzDE/psen_scan_v2/actions?query=event%3Apush+workflow%3ACI-Noetic+branch%3Amain) | [![CI-Foxy](https://github.com/PilzDE/psen_scan_v2/actions/workflows/ci_action_foxy.yml/badge.svg?branch=ros2)](https://github.com/PilzDE/psen_scan_v2/actions/workflows/ci_action_foxy.yml) | [![CI-Rolling](https://github.com/PilzDE/psen_scan_v2/actions/workflows/ci_action_rolling.yml/badge.svg?branch=rolling)](https://github.com/PilzDE/psen_scan_v2/actions/workflows/ci_action_rolling.yml) |
 | Buildfarm src | [![Build Status](http://build.ros.org/job/Msrc_uB__psen_scan_v2__ubuntu_bionic__source/badge/icon)](http://build.ros.org/job/Msrc_uB__psen_scan_v2__ubuntu_bionic__source/) | [![Build Status](http://build.ros.org/job/Nsrc_uF__psen_scan_v2__ubuntu_focal__source/badge/icon)](http://build.ros.org/job/Nsrc_uF__psen_scan_v2__ubuntu_focal__source/) | [![Foxy src build status](https://build.ros2.org/job/Fsrc_uF__psen_scan_v2__ubuntu_focal__source/badge/icon)](https://build.ros2.org/job/Fsrc_uF__psen_scan_v2__ubuntu_focal__source/) | |
 | Buildfarm bin | [![Build Status](http://build.ros.org/job/Mbin_uB64__psen_scan_v2__ubuntu_bionic_amd64__binary/badge/icon)](http://build.ros.org/job/Mbin_uB64__psen_scan_v2__ubuntu_bionic_amd64__binary/) | [![Build Status](http://build.ros.org/job/Nbin_uF64__psen_scan_v2__ubuntu_focal_amd64__binary/badge/icon)](http://build.ros.org/job/Nbin_uF64__psen_scan_v2__ubuntu_focal_amd64__binary/) | [![Foxy binary build status](https://build.ros2.org/job/Fbin_uF64__psen_scan_v2__ubuntu_focal_amd64__binary/badge/icon)](https://build.ros2.org/job/Fbin_uF64__psen_scan_v2__ubuntu_focal_amd64__binary/) | |
 
 
 ### Branching model
-`main` is considered to be the active development branch, it targets the ROS distributions `melodic` and `noetic`.
+[`main`](https://github.com/PilzDE/psen_scan_v2/tree/main) is considered to be the active development branch, it targets the ROS distributions `melodic` and `noetic`.
 
-The branch `ros2` is still experimental and targets the ROS 2 distributions `foxy` and `rolling`.
+The branch **ROS 2** branches [`foxy`](https://github.com/PilzDE/psen_scan_v2/tree/foxy) and [`rolling`](https://github.com/PilzDE/psen_scan_v2/tree/rolling) target the distributions according to their name.
```

Comment:
The branch `foxy` does not exist. In the moment we have `ros2` which only targets foxy and `rolling`. Some time ago I suggested to merge https://github.com/PilzDE/psen_scan_v2/pull/287, but I am not sure of the implications. It depends on which branching concept we want to keep in the future, if any.